perl/Parallel-ForkManager.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="https://apps.0install.net/perl/Parallel-ForkManager.xml" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
```

Comment:
Could you rename this feed to lower-case (`parallel-forkmanager.xml`)?

Reply:
lower cased